### DISCUSSION TITLE
Delete User In FireTV Client

### DISCUSSION THREAD

**Discussion by Russell-S** [jellyfin/jellyfin-androidtv#2449] [General]
Hi, I have been evaluating Jellyfin with a view to moving away from Plex which I use at the moment. I have the Jellyfin server running on Windows 10 and I predominantly use FireTV Stick as the clients on all the TV's. I have now set up all the users for the members of the household but I also have another user called 'Test' left over from when I was first setting it up. This user has been deleted from the server but is still present on the FireTV client.
I cannot find any way of removing that user from the login screen on the FireTV client. Is there any way of doing this without deleting the Jellyfin app completely and reinstalling and setting the users up again.

Thanks

**Comment by nielsvanvelzen** [comment]
Preferences (gear icon) -> Login -> Choose server -> Choose user -> Remove

**Reply by Russell-S** [reply]
That sorted it. Thanks very much.
